seacatauth/credentials/service.py

```diff
@@ -86,6 +86,25 @@ def __init__(self, app, service_name='seacatauth.CredentialsService', tenant_ser
 			self.register(provider)
 
 
+		# Metrics
+		self.MetricsService = app.get_service('asab.MetricsService')
+		self.Providers = providers
+		self.CredentialsGauge = self.MetricsService.create_gauge("credentials", tags={"help": "Counts credentials.", "unit": "users", "default_label":"SeaCatAuth Metrics"}, init_values={"credentials": 0})
+		app.PubSub.subscribe("Application.tick/10!", self._on_tick_metric)
+
+		
+	async def _on_tick_metric(self, event_name):
+		tasks = []
+		for order, provider in self.Providers:
+			tasks.append(provider.count())
+		counts = await asyncio.gather(*tasks)
+		total = 0
+		for count in counts:
+			if count != -1:
+				total += count
+		self.CredentialsGauge.set("credentials", total)
```

Comment:
As discussed, this should be done in the ASAB Task Service to off-load that from "on tick".